src/ui-shell/sidenav/side-nav.component.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { Component } from "@angular/core";
+import { TestBed, async } from "@angular/core/testing";
+import { By  } from "@angular/platform-browser";
+
+import { CommonModule } from "@angular/common";
+import { I18nModule } from "./../../i18n/i18n.module";
+import { SideNav } from "./sidenav.component";
+import { SideNavItem } from "./sidenav-item.component";
+import { SideNavMenu } from "./sidenav-menu.component";
+import { RouterModule } from "@angular/router";
+
+@Component({
+	selector: "app-foo",
+	template: "<h1>foo</h1>"
+})
+class FooComponent {}
+
+@Component({
+    template: `
+	<ibm-sidenav [allowExpansion]="allowExpansion" [hidden]="hidden">
+		<ibm-sidenav-menu title="Example Title"></ibm-sidenav-menu>
+		<ibm-sidenav-item
+			[route]="route"
+			(navigation)="onNavigation($event)">
+		</ibm-sidenav-item>
+	</ibm-sidenav>
+	`
+})
+class SideNavTest {
+	route = ['foo'];
+	hidden = false;
+	allowExpansion = false;
+	statusPromise = null;
+	onNavigation(event) {
+		this.statusPromise = event;
+	}
+}
+
+describe("SideNav", () => {
+	let fixture, wrapper, element;
+	beforeEach(() => {
+		TestBed.configureTestingModule({
+			declarations: [
+				SideNav,
+				SideNavItem,
+				SideNavMenu,
+				SideNavTest,
+				FooComponent
+			],
+			imports: [
+				CommonModule,
+				I18nModule,
+				RouterModule.forRoot([
+					{
+						path: "foo",
+						component: FooComponent
+					}
+				], {
```

Comment:
Seems like a more common way when listing things.
```suggestion
				],
				{
```